src/Illuminate/Database/Console/Migrations/MigrateCommand.php

```diff
@@ -16,7 +16,9 @@ class MigrateCommand extends BaseCommand
      */
     protected $signature = 'migrate {--database= : The database connection to use.}
                 {--force : Force the operation to run when in production.}
-                {--path= : The path of migrations files to be executed.}
+                {--once : Only run the first migration, so you can run them one by one.}
+                {--only= : Only run the specified migrations file.}
+                {--path= : The path to directory containing migrations files to be executed.}
```

Comment:
also updated wording on the --path help text, since when I first read "path" I think it implies the path to a file to run (hence adding the `--only=` option)